City-of-Helsinki/drupal-module-helfi-tpr
# UHF-2695: Allow service channels to be sorted

## How to install
- `composer require drupal/helfi_tpr:dev-UHF-2695`
-  `drush updb`

## How to test

- Publish a bunch of errand services and service channels then view errand service entities and make sure Channels are sorted in same order as defined in https://helsinkisolutionoffice.atlassian.net/browse/UHF-2695
- Change the default sort order from display settings (Go to Structure -> TPR- Errand Service settings -> Manage display, then click the settings cog on Channels and reorder the channel types). Then make sure the order is reflected in Errand service display
